api/v1beta2/ibmpowervsmachine_types.go

```diff
@@ -261,6 +283,22 @@ func (r *IBMPowerVSMachine) SetConditions(conditions capiv1beta1.Conditions) {
 	r.Status.Conditions = conditions
 }
 
+// GetV1Beta2Conditions returns the set of conditions for this object.
+func (r *IBMPowerVSMachine) GetV1Beta2Conditions() []metav1.Condition {
+	if r.Status.V1Beta2 == nil {
+		return nil
+	}
+	return r.Status.V1Beta2.Conditions
+}
+
+// SetV1Beta2Conditions sets conditions for an API object.
+func (r *IBMPowerVSMachine) SetV1Beta2Conditions(conditions []metav1.Condition) {
+	if r.Status.V1Beta2 == nil {
+		r.Status.V1Beta2 = &IBMPowerVSMachineV1Beta2Status{}
+	}
+	r.Status.V1Beta2.Conditions = conditions
+}
+
```

Comment:
@Karthik-K-N , where are we using this functions? I'm unable to find it in this PR, it is required for future changes?

Reply:
These are part of [Setter interface](https://github.com/kubernetes-sigs/cluster-api/blob/09e6c8a14c2dd955924e7534727974d1e89643e5/util/conditions/v1beta2/setter.go#L31-L37). We need to implement these interfaces to use utility functions to [set the conditions](https://github.com/Karthik-K-N/cluster-api-provider-ibmcloud/blob/da1d67b3f7a7a28e70e238c9c5866a44bae7588e/controllers/ibmpowervsmachine_controller.go#L268-L272), As they accept setter interface as parameter: https://github.com/kubernetes-sigs/cluster-api/blob/09e6c8a14c2dd955924e7534727974d1e89643e5/util/conditions/v1beta2/setter.go#L73

Reply:
Got it, Thanks @Karthik-K-N 👍🏼